Applications/Source/Palette Designer/Utilities/FilteredDescriptor.cs

```diff
@@ -0,0 +1,112 @@
+#region BSD License
+/*
+ *  New BSD 3-Clause License (https://github.com/Krypton-Suite/Standard-Toolkit/blob/master/LICENSE)
+ *  Modifications by Peter Wagner (aka Wagnerp), Simon Coghlan (aka Smurf-IV), Giduac, Ahmed Abdelhameed, tobitege et al. 2017 - 2025. All rights reserved.
+ */
+#endregion
+
+using System.ComponentModel;
```

Comment:
Minor, but can these be Alphabetically sorted please ?
Simple VS control KB sequence